tools/ci-trigger/pr.go

```diff
@@ -78,30 +81,62 @@ func (d *deviceType) String() string {
 	return d.Vendor.String() + " " + d.HardwareModel
 }
 
+// createArchive uploads the compressed repository to Object Store and returns the path to the object.
+func (p *pullRequest) createArchive(ctx context.Context, storClient *storage.Client) (string, error) {
```

Comment:
are these archives getting cleaned up? Not sure if this is a regression from the current impl

Reply:
They currently are not cleaned up, but my intent was to set an expiration policy on the gcp bucket to clean them up after 90d or so.

It is an improvement because we are going from one per vendor to only a single file, although bundling the go vendor deps is canceling out a lot of those savings.